package.json

```diff
@@ -34,10 +34,14 @@
     "node": ">=4"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0-beta.53",
+    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.53",
+    "@babel/preset-env": "^7.0.0-beta.53",
+    "@babel/preset-flow": "^7.0.0-beta.53",
     "babel-cli": "^6.18.0",
-    "babel-core": "^6.20.0",
+    "babel-core": "^6.26.3",
     "babel-eslint": "^8.2.3",
-    "babel-loader": "^6.2.9",
+    "babel-loader": "^7.1.5",
     "babel-plugin-external-helpers": "^6.22.0",
```

Comment:
We also don't need the external helpers plugin and the inline version anymore.

Reply:
Also, I think we should be able to remove babel-plugin-transform-es3-member-expression-literals, babel-plugin-transform-es3-property-literals, babel-plugin-transform-flow-strip-types and the old babel preset packages

Reply:
I know. I left everything for tests yet